apps/nextjs/src/components/AppComponents/Chat/AiSdk.tsx

```diff
@@ -69,8 +69,6 @@ export function AiSdk({ id }: Readonly<AiSdkProps>) {
     id,
     body: {
       id,
-      // NOTE: this lesson plan is used by the chat endpoint
-      lessonPlan,
```

Comment:
Can we delete line 49 above now? `const lessonPlan = useLessonPlanStore((state) => state.lessonPlan);`

Reply:
Indeed we can!